tests/test_build_latex.py

```diff
@@ -357,9 +357,9 @@ def test_numref_with_language_ja(app, status, warning):
     # sphinxmessages.sty
     result = (app.outdir / 'sphinxmessages.sty').text(encoding='utf8')
     print(result)
-    assert '\n{\\renewcommand{\\figurename}{図}}' in result
-    assert '\n{\\renewcommand{\\tablename}{表}}' in result
-    assert '\n{\\renewcommand{\\literalblockname}{リスト}}' in result
+    assert '\n\\@iden{\\renewcommand{\\figurename}{図}}' in result
+    assert '\n\\@iden{\\renewcommand{\\tablename}{表}}' in result
+    assert '\n\\@iden{\\renewcommand{\\literalblockname}{リスト}}' in result
```

Comment:
I used `\n` to test nothing exists before the braces. So it is no longer needed with this change.